src/test/java/Homework16.java

```diff
@@ -14,7 +14,7 @@ public static void RegistrationNavigation () {
         WebDriver driver = new ChromeDriver();
         driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
 
-        String url = "https://testpro.io/";
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example` public static void registrationNavigation ()`